### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ntBre**
## Description
Update the docs conda environment to build the docs successfully, and add a couple of examples/links to the docs themselves.

## Todos
  - [x] update env
  - [x] update docs

## Status
- [x] Ready to go

### CODE REVIEW

File: devtools/conda-envs/docs.yaml [hunk 1]
```diff
@@ -19,13 +19,14 @@ dependencies:
   - sphinxcontrib-bibtex
   - sphinx-notfound-page
   - myst-parser
+  - autodoc_pydantic
```
Comment (lilyminium) [inline devtools/conda-envs/docs.yaml:22]:
```suggestion
  - autodoc-pydantic
```

I think the spelling is different on conda! https://anaconda.org/conda-forge/autodoc-pydantic

Comment (ntBre) [reply devtools/conda-envs/docs.yaml]: Thanks Lily

**Review by j-wags — APPROVED** [review]
Nice @ntBre - Thanks for catching this and getting everything going again! The `inherited-members` change seems good for this method, though I don't have time this morning to see whether it's forwarding anything that we don't intend elsewhere in the docs (I think the odds are low, and internal folks are the primary consumers of these docs anyway, so we can just roll it back if something weird is happening).

Thanks!

File: devtools/conda-envs/docs.yaml [hunk 2]
```diff
@@ -19,13 +19,14 @@ dependencies:
...
+  - autodoc-pydantic
   - pandoc
   - ipython
 
     # Standard dependencies
-  - openff-toolkit-base >= 0.11.2
-#  - rdkit                MOCKED
-  - pydantic
+  - openff-toolkit-base =0.14.0
+  - rdkit                #MOCKED
```
Comment (j-wags) [inline devtools/conda-envs/docs.yaml:28]:
```suggestion
  - rdkit
```

**Pull request merged**